recipes/rnalien/build.sh

```diff
@@ -7,5 +7,8 @@ export CPPFLAGS="-I${PREFIX}/include"
 stack setup
 stack update
 stack install --extra-include-dirs ${PREFIX}/include --local-bin-path ${PREFIX}/bin
+mv ${PREFIX}/bin/RNAlien ${PREFIX}/bin/RNAlien-bin
```

Comment:
I think I will stay with the variant of writing the wrapper-script via echo for the moment :-D
Hope this is ok with you?